app/frontend/src/router/index.js

```diff
@@ -29,6 +29,11 @@ const router = new VueRouter({
           name: 'Dashboard',
           component: () => import(/* webpackChunkName: "dashboard" */ '@/views/admin/Dashboard.vue'),
         },
+        {
+          path: 'inspection-dashboard',
```

Comment:
Path seems a bit verbose IMHO but don't have any better suggestions. Could go /admin/dashboard/public and /admin/dashboard/inspection, but don't know if that's a good idea either.

Reply:
good suggestion. have followed your advice for now